php/libraries/User.class.inc

```diff
@@ -222,18 +248,16 @@ class User extends UserPermissions implements
             return $this->userInfo;
         } elseif ($var === 'CenterID') {
             throw new \LorisException(
-                "The function getData('CenterID')
-                                        is deprecated and is replaced
-                                        with getData('CenterIDs')"
+                "The function getData('CenterID') is deprecated and is 
+                replaced with getData('CenterIDs')"
             );
         } elseif ($var === 'Site') {
             throw new \LorisException(
-                "The function getData('Site')
-                                        is deprecated and is replaced
-                                        with getData('Sites')"
+                "The function getData('Site') is deprecated and is 
+                replaced with getData('Sites')"
             );
         } else {
-            return $this->userInfo[$var];
+            return $this->userInfo[$var] ?? null;
```

Comment:
Can you explain the reasoning behind this change?

Reply:
it's to prevent an error if the key doesn't exist in the userinfo.